packages/perennial-oracle/contracts/ChainlinkOracle.sol

```diff
@@ -27,7 +27,12 @@ contract ChainlinkOracle is IOracleProvider {
     int256 private immutable _decimalOffset;
 
     /// @dev Mapping of the first oracle version for each underlying phase ID
-    uint256[] private _startingVersionForPhaseId;
+    Phase[] private _phases;
```

Comment:
Can we update the comment for this, since now we're saving both the startingVersion and startingRound